src/test/run-pass-valgrind/unsized-locals/by-value-trait-objects-rust-call.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
remove the copyright header here and in all other new tests

Reply:
Sure, removed these headers for all unsized-locals tests including those from #51131.